src/Components/Footer/FooterElements.js

```diff
@@ -89,19 +89,19 @@ export const FooterLinkItems = styled.div`
 
 // Footer link title
 export const FooterLinkTitle = styled.h1`
-  font-size: 1.3rem;
-  font-weight: bold;
-
-  @media (min-width: 320px) {
-    margin-bottom: 0.5rem;
-  }
-
-  @media (min-width: 786px) {
-    margin-bottom: 1rem;
-  }
-`;
-
-// Footer link
+ font-size: 18px;
+ font-weight: bold;
+ margin-bottom: 16px;
+`
+export const FooterLinkTitleHr = styled.hr`
+  content: "";
+  border: 2px solid;
+  margin-top: 0;
+  width: 52%;
+  color:  #4AF60A;
```

Comment:
Could you change the color to this-- #01bf71

Reply:
@MonalikaPatnaik changed